src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -25,13 +26,13 @@ internal abstract partial class AbstractReferenceFinder : IReferenceFinder
     public const string ContainingMemberInfoPropertyName = "ContainingMemberInfo";
 
     public abstract Task<ImmutableArray<string>> DetermineGlobalAliasesAsync(
-        ISymbol symbol, Project project, CancellationToken cancellationToken);
+        FindReferencesSearchEngine searchEngine, ISymbol symbol, Project project, CancellationToken cancellationToken);
```

Comment:
the majority of this change in FindRefs is a mechanical passing of FindRefSearchEngine along (to grab the options or storage as needed).